qutebrowser/qutebrowser.py

```diff
@@ -115,23 +115,12 @@ def get_argparser():
     # For the Qt args, we use store_const with const=True rather than
     # store_true because we want the default to be None, to make
     # utils.qt:get_args easier.
-    debug.add_argument('--qt-name', help="Set the window name.",
-                       metavar='NAME')
     debug.add_argument('--qt-style', help="Set the Qt GUI style to use.",
                        metavar='STYLE')
```

Comment:
Ah, I see - I was trying to say the others are *never* needed, and this is only *almost* never needed :wink: I think it's fine to remove it too, as `--qt-arg style ...` can be used with this PR.